src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
-body {
+@import './utils/variables';
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
```

Comment:
Variables for 'gap' and 'padding' should be defined in the 'variables.scss' file according to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Ensure that these variables are properly declared there.